Helpers/OpenT2T-Wink-Helper/js/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+var https = require('https');
+var q = require('q');
+
+//this is our base, we refactor these options in each method
+
+
+var  accessToken,ApiPath;  //we will need this when creating the headers
+      
+module.exports = 
+{
+    //get the initial parameters to build our target endpoint
+   initWinkApi: function(apiEndpoint, deviceID, token)
+   {
+       ApiPath = '/' + apiEndpoint + '/' + deviceID;
+       console.log("Api path is " + ApiPath)
+       accessToken = token; //the specific token
+        
+   },
+ 
+   sendDesiredStateCommand: function(apiField,value) 
+   {
+       
+    var options = 
+        {
```

Comment:
Please make the indentation consistent throughout this file (4 spaces). Currently various blocks are using 1, 2, 3, or 4 spaces.